src/Compilers/CSharp/Test/Symbol/Compilation/SemanticModelGetDeclaredSymbolAPITests.cs

```diff
@@ -1999,10 +1999,10 @@ static void M()
             var compilation = CreateCompilationWithMscorlib(text);
             var tree = compilation.SyntaxTrees.Single();
             var model = compilation.GetSemanticModel(tree);
-            var position = text.IndexOf("{", text.IndexOf("a)"));
+            var position = text.IndexOf('{', text.IndexOf("a)", StringComparison.Ordinal));
```

Comment:
When a char argument is provided (rather than a single-char string literal), it is effectively equivalent to providing StringComparison.Ordinal, but ~5 time faster.